documentation/releases.md

```diff
@@ -4,11 +4,14 @@
 
 | Version | Original Release Date | Latest Patch Version | Patch Release Date | End of Support | Runtime Frameworks |
 |---|---|---|---|---|---|
-| 6.1 | February 17, 2022 | [6.1.1](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/releaseNotes/releaseNotes.v6.1.1.md) | April 12, 2022 |  | .NET Core 3.1 (with major roll forward)<br/>.NET 6 |
+| 6.2 | June 14, 2022 | [6.2](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/releaseNotes/releaseNotes.v6.2.0.md) | June 14, 2022 |  | .NET Core 3.1 (with major roll forward)<br/>.NET 6 |
+| 6.1 | February 17, 2022 | [6.1.2](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/releaseNotes/releaseNotes.v6.1.2.md) | June 14, 2022 |  | .NET Core 3.1 (with major roll forward)<br/>.NET 6 |
+| 6.1 | February 17, 2022 | [6.1.1](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/releaseNotes/releaseNotes.v6.1.1.md) | April 12, 2022 | July 12th  | .NET Core 3.1 (with major roll forward)<br/>.NET 6 |
```

Comment:
Please remove 6.1.1; we should only keep the latest for each major-minor version.